index.html

```diff
@@ -39,43 +42,102 @@ <h3 class="font-bold mb-2 text-xl">
             Firefox-lisäosista.
           </p>
           <p class="mb-2">
-            <a class="underline hover:bg-yellow-100" href="#">Jaksomuikkarit</a>
+            <a class="italic underline hover:text-red-400" href="#"
+              >Jaksomuikkarit</a
+            >
           </p>
           <p><em>Player</em></p>
         </article>
 
         <h2 class="font-bold mb-4 text-xl">Menneet jaksot</h2>
 
-        <article class="-m-2 p-2 rounded hover:bg-yellow-100">
-          <a href="#">
-            <h3 class="font-bold mb-2 text-lg underline">#2 &ndash; ?</h3>
-            <footer>
-              <p class="mb-2">
-                <span class="sr-only">Julkaistu</span>
-                <time datetime="2021-03-27">27.3.2021</time>
-              </p>
-            </footer>
-            <p>Jaksokuvaus...</p>
-          </a>
-        </article>
+        <div class="flex flex-wrap">
+          <article class="w-full md:w-2/4 mb-3">
+            <div
+              class="mr-3 rounded bg-white shadow-md hover:shadow-lg transition group"
+            >
+              <a href="#" class="block p-3">
```

Comment:
Note for later:

Dunno if it's confusing for screen reader users that the `<a>` tag contains so much stuff. We could move it under the `<h3>` and use CSS to make the whole block clickable. (So that the `<a>` has a pseudo element (`:before` or `:after`) that covers the whole block.)